js/adapt-assessmentArticleView.js

```diff
@@ -29,6 +30,26 @@ define([
             this.stopListening(Adapt, "assessments:reset", this._onAssessmentReset);
         },
 
+        setupQuestionNumbering: function() {
+            Handlebars.registerHelper("questionNumber", function getQuestionNumber() {
+                var currentArticle = this._parent.attributes._parentId;
```

Comment:
There are built-in APIs which you can leverage to grab attributes like this. For example, I'd obtain the model through `this.view.model` and use
```js
model.findAncestor('articles').get('_id')
```
Might be an idea to reference the fact it's an ID in the variable name for clarity.